.travis.yml

```diff
@@ -41,6 +42,9 @@ matrix:
     env: INSTALL=gcc
     compiler: gcc-6
     osx_image: xcode8
+  - os: osx
+    compiler: clang
+    osx_image: xcode9
```

Comment:
Drive-by feature creeper or more proof of the awesome cross-distro support? 
- also added `Mac OS Xcode 9` testing support